test/src/waittask.spec.ts

```diff
@@ -407,6 +407,31 @@ describe('waittask specs', function () {
       await watchdog;
     });
 
+    // MutationPoller currently does not support shadow DOM.
```

Comment:
Let's add the spec issue here as well of safe keeping.

Reply:
I think it is not directly related to MutationPoller since it is our own abstraction and we can probably make it work without MutationObserver supporting shadow DOM out of the box.